pyfakefs/tests/pytest/pytest_plugin_test.py

```diff
@@ -1,7 +1,24 @@
 """Tests that the pytest plugin properly provides the "fs" fixture"""
 import os
+import tempfile
 
 
 def test_fs_fixture(fs):
     fs.create_file('/var/data/xx1.txt')
     assert os.path.exists('/var/data/xx1.txt')
+
+
+def test_pause_resume(fs):
+    fake_temp_file = tempfile.NamedTemporaryFile()
+    assert fs.exists(fake_temp_file.name)
+    assert os.path.exists(fake_temp_file.name)
+    fs.pause()
+    assert fs.exists(fake_temp_file.name)
+    assert not os.path.exists(fake_temp_file.name)
+    real_temp_file = tempfile.NamedTemporaryFile()
+    assert not fs.exists(real_temp_file.name)
+    assert os.path.exists(real_temp_file.name)
+    fs.resume()
+    assert not os.path.exists(real_temp_file.name)
+    assert os.path.exists(fake_temp_file.name)
+
```

Comment:
Do `resume()` again, expect an exception.

Reply:
Actually I strongly agree.  The state before the invocation is irrelevant as long as it's paused after `pause()` and resumed after `resume()`.  The API should always be permissive in this way.

Reply:
Ok, glad that we are on the same page - though it took me an iteration to come to that.
Are there any changes you want to see?